### PR TITLE
ft: search functionality

### PR DESCRIPTION
**Pull request by sticklo**
searchbar display feature fix

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
...
+    setSearchState(value);
+  };
+
   return (
     <div className="App">
-      <Home />
+      <Home onSearch={handleSearch} />
+      {searchState ? <Search /> : null}
     </div>
   );
```
Comment (Asylcreek) [inline client/src/App.js:18]: Can you please explain what is happening here?

File: client/src/index.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
```
Comment (Asylcreek) [inline client/src/index.css:2]: This import is already in the index.html file, please check and confirm. If it's there, please remove this import

File: client/src/App.js (hunk 1 above)
Comment (sticklo) [reply client/src/App.js:18]: I'm saving the stage of the search bar to show when to show it and when to hide it. I want the search bar to disappear when the user clicks on any other part of the page

File: client/src/index.css (hunk 2 above)
Comment (sticklo) [reply client/src/index.css:2]: solved

File: client/src/App.js (hunk 1 above)
Comment (sticklo) [reply client/src/App.js:18]: line 7: I  initialized the state and got a function to update the state from use state hook
                                                                    
line 8 & 9: is a method to handle the search input changing, I'm getting the value from the on-change event of the input and setting the search state to whatever the value is ; that value is gotten conditionally i.e  if an event of blur happens  , it updates the state to empty strings, that means the user left the search input. but if the type of the event is not the blur event, I want to get the value of whatever is in the search input 

line 10:  it to update the state to whatever values 

line 15: is adding a prop of onSearch  to the function created on line 8

line 16:  is  rendering the search component conditionally if the search state is truthy (if it has a value); if it is an empty string (falsy)  it is not going to render any value

**Comment by KelvinAkpobome** [comment]
Please resolve conflicts

**Comment by KelvinAkpobome** [comment]
Please resolve conflicts

**Pull request merged**